sdk/nodejs/automation/stack.ts

```diff
@@ -118,7 +119,13 @@ export class Stack {
             })
             .pipe(split2())
             .on("data", (line: string) => {
-                const event: EngineEvent = JSON.parse(line);
+                let event: EngineEvent;
+                try {
+                    event = JSON.parse(line);
+                } catch (e) {
+                    log.info(`failed to parse engine event\nevent: ${line}\n${e.toString()}`);
```

Comment:
Isn't an `await` missing here? 

```
await log.info(`failed to parse engine event\nevent: ${line}\n${e.toString()}`);
```

Reply:
nope, that function doesn't return a promise